Mathlib/Topology/Algebra/InfiniteSum/Nonarchimedean.lean

```diff
@@ -55,6 +55,39 @@ theorem cauchySeq_prod_of_tendsto_cofinite_one {f : α → G} (hf : Tendsto f co
   intro i hi
   simpa using Finset.disjoint_left.mp ht hi
 
+/-- Let `G` be a nonarchimedean multiplicative abelian group, and let `f : ℕ → G` be a function
+such that the quotients tend to one. Then the function is a Cauchy sequence. -/
```

Comment:
Please spell out "quotients `f (n + 1) / f n`", otherwise it's not very clear (& likewise for the additive docstring). Also I think "multiplicative" is redundant, groups in Mathlib are multiplicative unless stated otherwise.